deploy/ods-pipeline/charts/tasks/templates/task-ods-package-image.yaml

```diff
@@ -30,11 +30,11 @@ spec:
     - name: dockerfile
       description: Path to the Dockerfile to build (relative to `docker-dir`).
       type: string
-      default: ./Dockerfile
+      default: ./docker/Dockerfile
```

Comment:
I would tend to keep `./Dockerfile`. I know this is a breaking change but  changing the `docker-dir` already is one.